File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.js
function solution(absolutes, signs) {
    let result = 0;
    for(let i=0; i<signs.length; i++) {
        if(signs[i] === true) {
            absolutes[i] = Math.abs(absolutes[i]);
        } else {
            absolutes[i] = absolutes[i]*-1;
        }
    }
    for(let j of absolutes) {
        result += j;
    }
    return result;
}

File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;
        for(int i=0; i<signs.length; i++) {
            if(signs[i] == true) {
                absolutes[i] = Math.abs(absolutes[i]);    
            } else {
                absolutes[i] = absolutes[i] * -1;
            }
        } 
        for(int i=0; i<absolutes.length; i++) {
            answer += absolutes[i];
        }
        return answer;
    }
}